contracts/TokenRequest.sol

```diff
@@ -131,7 +131,7 @@ contract TokenRequest is AragonApp {
     }
 
     /**
-    * @notice Create a token request depositing `@tokenAmount(_depositToken, _depositAmount, true, _depositToken.decimals(): uint256)` in exchange for `@tokenAmount(self.getToken(): address, _requestAmount, true, 18)`
+    * @notice Create a token request depositing `@tokenAmount(_depositToken, _depositAmount, true, 18)` in exchange for `@tokenAmount(self.getToken(): address, _requestAmount, true, 18)`
```

Comment:
Note that changed to 18 precision decimals since the deposited token could be ETH resulting in the function call `_depositToken.decimals()` not working. The current version of radspec sets 2 as default precision if not given. When the new version for radspec  is released, we should remove this values since there's an enhancement where if not given the precision value, it takes the token decimals by default. 
See https://github.com/aragon/radspec/pull/78